generated/.swagger-codegen-ignore

```diff
@@ -0,0 +1,23 @@
+# Swagger Codegen Ignore
```

Comment:
Useless file.

Reply:
fixed in 09b5f9cd6b08f41f9226ab61decbf54bb237b99b